lib/src/Uncertainty/Distribution/cephes/dd_real.h

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
So with this, we embed an extended precision library in addition to the optional mpfr/mpc multiprecision libraries used in SpecFunc. At one point we should coordinate this, in particular we could make it more visible troughout the whole library eg. to implement roundoff sensitive algorithms (orthonormalization, dot products...)

Reply:
yes, I did not want to rewrite that part